mdsuite/database/calculator_database.py

```diff
@@ -268,7 +268,7 @@ def _get_rdf_data(self) -> List[db.Computation]:
         """Fill the data_files list with filenames of the rdf tensor_values"""
         # TODO replace with exp.load.RDF()
         raise DeprecationWarning(
-            "Replaced by experiment.run.RadialDistributionFuncion(**kwargs)"
+            "Replaced by experiment.run.RadialDistributionFunction(**kwargs)"
         )
```

Comment:
is this still used after the refactor? I think we can remove this entirely

Reply:
I dont know Im gonna leave it for now.